### PR TITLE
Adds a new NCR role, the Provost Marshal.

### PR DESCRIPTION
**Pull request by Sinbest**
## About The Pull Request

In this pull request, I am adding a new role to lead the military police, and some other minor changes that will be mentioned in the changelog.

The Provost Marshal's rank is 2st Lt. For equipment, they get a riot shotgun and 44. revolver.

## Why It's Good For The Game

Why is the Provost Marshal needed?

- The Military Police need leadership.
- The Military Police need the authority of an officer backing them up, since most of the NCR ignores their attempts to enforce military law and discipline for nonsensical reasons.
- Rogue officers and sergeants often weaponize the rest of the faction to avoid being dealt with.

An MP who is also an officer could help resolve all of these problems.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Added the Provost Marshal role
add: Added the Provost Marshal landmark to the map so they spawn inside the base
tweak: MPs get Staff sergeant rank pins
tweak: increased MP slots from 2 to 3
tweak: The Colonel and his assistant get leather satchels to match their dress uniforms
/:cl:

### CODE REVIEW

**Comment by SeeminglyTypicalUsername** [comment]
This is quite good.

**Comment by BadAtThisGame302** [comment]
Why is this needed? Why more bloat roles for the NCR?

**Comment by Sinbest** [comment]
After thinking about it, I'll bump the Marshal's rank down to 2nd LT.

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -717,7 +761,7 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	head = /obj/item/clothing/head/f13/ranger
 	uniform	= /obj/item/clothing/under/f13/ranger/patrol
 	belt = /obj/item/storage/belt/military/assault/ncr
-	suit_store = /obj/item/gun/ballistic/automatic/service/carbine
+	suit_store = /obj/item/gun/ballistic/automatic/assault_rifle
```
Comment (Kitsunemitsu) [inline code/modules/jobs/job_types/ncr.dm:764]: Irrelevant to the PR, Remove.

File: code/modules/jobs/job_types/ncr.dm [hunk 2]
```diff
@@ -702,9 +746,9 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	suit = /obj/item/clothing/suit/armor/f13/trailranger
 	belt = /obj/item/storage/belt/military/NCR_Bandolier
 	neck = /obj/item/clothing/neck/mantle/ranger
-	suit_store = /obj/item/gun/ballistic/rifle/repeater/trail
+	suit_store = /obj/item/gun/ballistic/automatic/rangemaster
```
Comment (Kitsunemitsu) [inline code/modules/jobs/job_types/ncr.dm:749]: Irrelevant to the PR. Stop stealth-buffing rangers

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
@ogrigbe This PR has some serious issues, can you please not merge until you look at all the stealth buffs I've outlined

File: code/modules/jobs/job_types/ncr.dm [hunk 3]
```diff
@@ -232,8 +233,7 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	selection_color = "#fff5cc"
 	display_order = JOB_DISPLAY_ORDER_LIEUTENANT
 	outfit = /datum/outfit/job/ncr/f13lieutenant
-	exp_type = EXP_TYPE_NCR
-	exp_requirements = 1000
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:236]: LT timelock change you did too.

File: code/modules/jobs/job_types/ncr.dm [hunk 4]
```diff
@@ -1174,9 +1217,9 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 
 /datum/outfit/loadout/trooperfiresupport
 	name = "Sapper"
-	suit_store = /obj/item/gun/ballistic/automatic/m1carbine/m1n
+	suit_store = /obj/item/gun/ballistic/shotgun/hunting
 	backpack_contents = list(
-		/obj/item/ammo_box/magazine/m10mm_adv/ext = 2,
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:1179]: Random loadout changes too.

File: code/modules/jobs/job_types/ncr.dm [hunk 5]
```diff
@@ -1387,7 +1430,7 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	description = "You are the support element sent to assist the Camp Miller garrison. You are essential specialized support staff to help sustain the base via supply or specialized skills. You are not allowed to leave base unless given an explicit order by the CO or the current acting CO."
 	supervisors = "Logistics/Medical officer first, regular chain of command after that."
 	selection_color = "#fff5cc"
-	exp_type = EXP_TYPE_FALLOUT
+	exp_type = EXP_TYPE_NCR
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:1433]: LO timelock change.

File: code/modules/jobs/job_types/ncr.dm [hunk 6]
```diff
@@ -384,8 +431,7 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	access = list(ACCESS_NCR, ACCESS_NCR_ARMORY, ACCESS_NCR_COMMAND)
 	display_order = JOB_DISPLAY_ORDER_SERGEANT
 	outfit = /datum/outfit/job/ncr/f13drillsergeant
-	exp_type = EXP_TYPE_NCR
-	exp_requirements = 1500
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:388]: SEA timelock changes.

**Comment by Sinbest** [comment]
Github glitched, fixing it now.

**Review by BadAtThisGame302 — CHANGES_REQUESTED** [review]

**Comment by Sinbest** [comment]
Closed due to a glitch which ported outdated code, making people think I'm sneaking things in.
Will remake the entire PR at a later date when the resulting anger over the glitch cools down.

**Pull request closed without merging**